src/masternodes/govvariables/lp_daily_dfi_reward.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2020 The DeFi Foundation
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <masternodes/govvariables/lp_daily_dfi_reward.h>
+
+#include <core_io.h> /// ValueFromAmount
+#include <rpc/util.h> /// AmountFromValue
+
+
+Res LP_DAILY_DFI_REWARD::Import(const UniValue & val) {
+    dailyReward = AmountFromValue(val);
+    return Res::Ok();
+}
+
+UniValue LP_DAILY_DFI_REWARD::Export() const {
+    return ValueFromAmount(dailyReward);
+}
+
+Res LP_DAILY_DFI_REWARD::Validate(const CCustomCSView &) const
+{
+    // nothing to do
+    return Res::Ok();
```

Comment:
Why don't need Validate here?

Reply:
For example? I have no any criteria to validate "daily dfi reward". <0? >MAX_MONEY? 
It comes from "foundation" only, and firstly checked with auth on apply. Basic ranges was checked in import (called from rpc) with AmountFromValue.
One of foundation members will cheat, manually constructing tx bytecode to avoid rpc's AmountFromValue to place into tx negative value?

Reply:
Another words, the general validation of "setgov" is an auth. All others are only aimed at protecting "the gods" from hurtful mistakes (as with total sum of 100% in LP_SPLITS, or pool existence)